packages/manager/apps/pci-instances/src/data/hooks/instance/action/useInstanceBackupPrice.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  convertHourlyPriceToMonthly,
+  priceFromUcent,
+} from '@ovh-ux/manager-react-components';
+import { useProductAvailability, useCatalog } from '@ovh-ux/manager-pci-common';
+import { useTranslation } from 'react-i18next';
+import { useMemo } from 'react';
+
+export const useInstanceBackupPrice = (projectId: string, region: string) => {
+  const { i18n } = useTranslation('actions');
+  const locale = i18n?.language?.replace('_', '-');
```

Comment:
Are the null check mandatory ?

Reply:
Agree, optional chainings should be removed